tests/source/disable_all_formatting.rs

```diff
@@ -0,0 +1,4 @@
+// rustfmt-disable_all_formatting: true
```

Comment:
This test doesn't do anything because there is not corresponding file in tests/target. However, since you only want to do an idempotent test, you could move it to tests/target and it would be fine.